yt_dlp/extractor/vk.py

```diff
@@ -701,3 +704,139 @@ def _real_extract(self, url):
+            'like_count': int,
+            'categories': ['Atomic Heart'],
+        },
+        'params': {'skip_download': 'm3u8'},
+    }]
+
+    def _real_extract(self, url):
+        username, video_id = self._match_valid_url(url).groups()
+
+        initial_state = self._extract_initial_state(url, video_id)
+        record_info = traverse_obj(initial_state, ('record', 'currentRecord', 'data', {dict}))
+        if not record_info:
+            record_info = self._download_json(
+                f'https://api.vkplay.live/v1/blog/{username}/public_video_stream/record/{video_id}', video_id)
+
+        return {
+            'id': video_id,
+            'formats': self._extract_formats(record_info, video_id),
+            **self._extract_common_meta(record_info),
+            **traverse_obj(record_info, {
+                'uploader': ('blog', 'owner', 'nick', {str_or_none}),
+                'uploader_id': ('blog', 'owner', 'id', {int_or_none}),
+                'duration': ('duration', {int_or_none}),
+            })
+        }
+
+
+class VKPlayLiveIE(VKPlayBaseIE):
+    _VALID_URL = r'https?://vkplay\.live/(?P<id>[^/]+)(?:[#?]|$)'
+    _TESTS = [{
+        'url': 'https://vkplay.live/bayda',
+        'info_dict': {
+            'id': 'f02c321e-427b-408d-b12f-ae34e53e0ea2',
+            'ext': 'mp4',
+            'title': r're:эскапизм крута .*',
+            'uploader': 'Bayda',
+            'uploader_id': 12279401,
+            'release_timestamp': 1687209962,
+            'release_date': '20230619',
+            'thumbnail': r're:https://images.vkplay.live/public_video_stream/12279401/preview\?change_time=\d+',
+            'view_count': int,
+            'concurrent_view_count': int,
+            'like_count': int,
+            'categories': ['EVE Online'],
+            'live_status': 'is_live',
+        },
+        'skip': 'livestream',
+        'params': {'skip_download': True},
+    }]
+
+    def _real_extract(self, url):
+        username = self._match_id(url)
+
+        initial_state = self._extract_initial_state(url, username)
+        stream_info = traverse_obj(initial_state, ('stream', 'stream', 'data', 'stream', {dict}))
+        if not stream_info:
+            stream_info = self._download_json(
+                f'https://api.vkplay.live/v1/blog/{username}/public_video_stream', username)
+        formats = self._extract_formats(stream_info, username)
+        if not formats and not traverse_obj(stream_info, ('isOnline', {bool})):
+            raise UserNotLive(video_id=username)
+
+        return {
+            'formats': formats,
+            **self._extract_common_meta(stream_info),
+            **traverse_obj(stream_info, {
+                'id': ('id', {str}),
+                'uploader': ('user', 'nick', {str_or_none}),
+                'uploader_id': ('user', 'id', {int_or_none}),
+                'concurrent_view_count': ('count', 'viewers', {int_or_none}),
+                'is_live': ('isOnline', {bool}),
```

Comment:
`is_live` could just be returned as `True` since we already check its value and raise `UserNotLive` if false